docker-compose.yaml

```diff
@@ -0,0 +1,20 @@
+# To customize this, add environment options in .env
+# see .env.example for example
+
+version: "3.9"
+services:
+  agent:
+    image: portainer-agent
+    container_name: portainer_edge_agent_dev
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - /var/lib/docker/volumes:/var/lib/docker/volumes
+      - /usr/share/zoneinfo:/usr/share/zoneinfo
```

Comment:
what's this? timezone?

Reply:
Because I was running portainer outside of a container and the agent ran inside a container it meant that they were running with different timezone information.   Therefore when you pushed an update script it wouldn't run.